Makefile

```diff
@@ -54,12 +54,12 @@ LDFLAGS += -X $(GO_PROJECT)/internal/util.DriverVersion=$(CSI_IMAGE_VERSION)
 
 BASE_IMAGE ?= $(shell . $(CURDIR)/build.env ; echo $${BASE_IMAGE})
 
-ifndef CEPH_VERSION
-	CEPH_VERSION = $(shell . $(CURDIR)/build.env ; echo $${CEPH_VERSION})
+ifndef BUILD_TAGS
+	BUILD_TAGS = $(shell . $(CURDIR)/build.env ; echo $${BUILD_TAGS})
```

Comment:
may be call this as `CEPH_BUILD_TAGS" as it is referring to Ceph Build ?

Reply:
We do not need to prefix everything with `CEPH_CSI_` or `CEPH_` imho. `BUILD_TAGS` suggest any Golang build-tags can be included, which is the goal here.